js/main.js

```diff
@@ -0,0 +1,83 @@
+/* global movieData */
+
+const $submit = document.querySelector('#film-search');
+const $searchTerm = document.querySelector('#film-search-input');
+
+$submit.addEventListener('submit', captureSearch);
+let term = '';
```

Comment:
If you're not going to use this variable in any other function besides `captureSearch`, then you should move the initialization inside `captureSearch`

Reply:
I think I might, but I'll move it back into the function, just in case.